src/styles/pages/games/Wordle.css

```diff
@@ -0,0 +1,134 @@
+.Wordle {
+    text-align: center;
+    background-color: black;
+    width: 100vw;
+    height: 100vh;
+    /* display: flex; */
+    color: white;
+  }
+  
+  body {
```

Comment:
dont' add css to `body`, `h1`, `div` `p` and similar tags that can affect other components as well.